README.md

```diff
@@ -304,6 +305,13 @@ scrape({
 Number, maximum amount of concurrent requests. Defaults to `Infinity`.
 
 
+#### replacelinks
+Boolean. Defaults to `true`. Sometimes you may want to use `false` if
```

Comment:
I think it is not necessary to be related with filename generator. 
I'd suggest just simple description like `If true - set local paths as links for downloaded sources, if false - do not update links, leave them as is`

As for **but it does not always work** - I think it would be better to fix the reason instead of saying in documentation that it does not work :) could you please provide examples for such problems?